app/client/components/delivery/address/deliveryAddressDisplay.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {
+  DeliveryAddress,
+  ProductDetail
+} from "../../../../shared/productResponse";
+import { ProductUrlPart } from "../../../../shared/productTypes";
+import { LinkButton } from "../../buttons";
+
+interface DeliveryAddressDisplayProps extends DeliveryAddress {
+  withEditButton: boolean;
```

Comment:
```suggestion
  withEditButton?: true;
```
... so then the usage could simply be `withEditButton` in the JSX usage, rather than `withEditButton={true}`